stack/cas/installhelper.class.php

```diff
@@ -29,7 +29,7 @@ class stack_cas_configuration {
     protected static $instance = null;
 
     /** @var array This variable controls which optional packages are supported by STACK. */
-    public static $maximalibraries = array('stats', 'distrib', 'descriptive', 'simplex');
+    public static $maximalibraries = array('stats', 'distrib', 'descriptive', 'simplex', 'bitwise');
```

Comment:
Is needed for the base n work, or just another useful library of Maxima functions you'd like the option of having?!

I've reviewed this, and it looks like a sensible inclusion, so I'm happy to have it here.  If the base n work needs it then we should just load it automatically, otherwise it will cause confusion if people don't have this, and then things break.